Source/Evolve/WarpXEvolve.cpp

```diff
@@ -249,7 +250,9 @@ WarpX::Evolve (int numsteps)
             // positions are next pushed) so that the particles do not deposit out of bounds
             // and so that the fields are at the correct time in the output.
             bool const reset_fields = true;
-            ComputeSpaceChargeField( reset_fields );
+            if (warpx_py_fieldsolver) warpx_py_fieldsolver();
+                else AddSpaceChargeFieldLabFrame();
```

Comment:
Indentation needs to be fixed - neither the else nor the line below should be further indented.